### PR TITLE
doc: create 0.1.22 changelog

### PR DESCRIPTION
**Pull request by sergos**

### CODE REVIEW

**Comment by Serpentian** [comment]
Let's add a message about service_info, move API changes to the corresponding part of the changelog and fix a typo in `intorudced`:

```diff
diff --git a/changelogs/0.1.22.md b/changelogs/0.1.22.md
index 5757a6d..36c053a 100644
--- a/changelogs/0.1.22.md
+++ b/changelogs/0.1.22.md
@@ -6,10 +6,16 @@ Tag: 0.1.22
 
 VShard 0.1.22 is fully compatible with the previous VShard versions.
 
+## Functionality added or changed
+
+* A new vshard.storage.on_bucket_event() event API to create a user reaction to
+  a bucket move is introduced (gh-372).
+
+* vshard.router/storage.info() now accepts options, which can be used to get
+  additional information about background services (such as discovery or
+  rebalancer) that are working on the current instance (gh-107).
+
 ## Bugs fixed
 
 * Fixed a bug that the rebalancing can lead to inconsistency in buckets (gh-377,
   gh-173).
-
-* A new vshard.storage.on_bucket_event() event API to create a user reaction to 
-  a bucket move is intorudced (gh-372).

```

File: changelogs/0.1.22.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Date: 20-02-2022
+
+Tag: 0.1.22
```
Comment (Gerold103) [inline changelogs/0.1.22.md:3]: There are more changes:
- https://github.com/tarantool/vshard/commit/926945756dfc503ca71cb5b36dcf44d918a8c9b4 and https://github.com/tarantool/vshard/commit/b6b95d0ec4da68da16abfcf0d9fe24e6cf58aa04;
- https://github.com/tarantool/vshard/commit/b47b6c29a7bf5e63af89138730402bab34965a67;
- https://github.com/tarantool/vshard/commit/184f640f6f4ee8cecbd6ec17ec7bcd5c6be3db28;
- https://github.com/tarantool/vshard/commit/b01586599f063df1145b48104241a3a9f3e699de;
- https://github.com/tarantool/vshard/commit/d9b2455a741f4096b2ac06fe50aa9afb2ded0c6a;
- https://github.com/tarantool/vshard/commit/0cb510237c1b787472846240db43369e4861c22d and https://github.com/tarantool/vshard/commit/f5f386a5a35e6e5efd8f4f2ed1b3d208fdae9095;

You can see the full list here: https://github.com/tarantool/vshard/compare/0.1.21...master.

**Review by Gerold103 — CHANGES_REQUESTED** [review]
Thanks for the patch!

**Comment by Gerold103** [comment]
Please, use this:
```
Date: 20-02-2023

Tag: 0.1.22

## Compatibility

VShard 0.1.22 is fully compatible with the previous VShard versions.

## Functionality added or changed

* A new trigger `vshard.storage.on_bucket_event()` to react on bucket events,
  such as move or GC (gh-372).

* `vshard.router/storage.info()` now accept an option `{with_services = true}`
  to get additional info about background services (such as discovery or
  rebalancer) that are working on the current instance (gh-107).

## Bugs fixed

* Fixed a bug that the rebalancing could lead to inconsistency in buckets. In
  scope of the same bugfix all the updates in `_bucket` space are now severely
  restricted and validated (gh-377, gh-173).

* `vshard.router/storage.cfg{}` now can not be called from multiple fibers at
  the same time (gh-140).

* Fixed a bug that certain router error objects didn't have `router_name` field.
```

Some of the commits I listed in the review above I omitted as not documented (on_master_enable/disable triggers) or not released bugs (bucket on_rollback was introduced and fixed after 0.1.21).

File: changelogs/0.1.22.md (hunk 1 above)
Comment (sergos) [reply changelogs/0.1.22.md:3]: - #140 didn't have any real cases - so I don't think it's reasonable to put 9269457 and b6b95d0 here.
- b47b6c2  has no associated issue and the behavior is named 'undocumented'. Added anyways.
- 184f640 #173 is mentioned in the rebalancing inconsistency fix 
- b015865 ditto
- d9b2455 mentioned as #107 
- 0cb5102 and f5f386a ditto

**Comment by sergos** [comment]
Done as per your proposal.

**Review by Gerold103 — CHANGES_REQUESTED** [review]
Checkpatch CI is failing.

**Pull request merged**